lib/dialects/abstract/query-generator.js

```diff
@@ -21,7 +20,7 @@ const QueryGenerator = {
     tableName = tableName || {};
     return {
       schema: tableName.schema || options.schema || 'public',
-      tableName: _.isPlainObject(tableName) ? tableName.tableName : tableName,
+      tableName: Utils._.isPlainObject(tableName) ? tableName.tableName : tableName,
```

Comment:
yes. In many cases you don't even need lodash because theres a vanilla JS equivalent for it now